web/src/OverviewResourceDetails.tsx

```diff
@@ -13,29 +18,161 @@ let OverviewResourceDetailsRoot = styled.div`
   flex-direction: column;
 `
 
-let ActionBar = styled.div`
+let CopyButtonRoot = styled.button`
+  font-family: ${Font.sansSerif};
+  display: flex;
+  align-items: center;
+  padding: 8px 12px;
+  margin: 0;
+
+  background: ${Color.grayDark};
+
+  border: 1px solid ${Color.grayLighter};
+  box-sizing: border-box;
+  border-radius: 4px;
+  cursor: pointer;
+  transition: color 300ms ease, border-color 300ms ease;
+  color: ${Color.gray7};
+
+  & .fillStd {
+    fill: ${Color.gray7};
+    transition: fill 300ms ease;
```

Comment:
Minor point, but consider using the ${AnimDuration.default} helper?

Reply:
done!